packages/manager/modules/netapp/src/dashboard/network-configuration/controller.js

```diff
@@ -85,12 +85,10 @@ export default class OvhManagerNetAppNetworkConfigurationCtrl {
       this.selectedSubnet = noSubnet;
       this.disableSubnetField = true;
     } else {
-      this.subnets.forEach((subnet) => {
-        if (!subnet.displayName) {
-          // eslint-disable-next-line no-param-reassign
-          subnet.displayName = subnet.cidr;
-        }
-      });
+      this.subnets = this.subnets.map((subnet) => ({
+        ...subnet,
+        displayName: subnet.displayName ? subnet.displayName : subnet.cidr,
+      }));
```

Comment:
It's a bit weird to override your object reference. Could it be done directly in 
```  
this.subnets = angular.copy(vrackService.currentState.subnets).map((subnet) => ({
        ...subnet,
        displayName: subnet.displayName ?? subnet.cidr,
      }));
``` 
?

Reply:
Moreover, on existing code ` this.subnets.push(noSubnet);` could fail if `this.subnets `is null